internal/advtools/shooting.lua

```diff
@@ -0,0 +1,28 @@
+-- Fixes shooting/throwing options stacking on each other, causing null pointers and crashes
+--@ module = true
+
+local aim_projectile = df.global.game.main_interface.adventure.aim_projectile
+
+local overlay = require('plugins.overlay')
+
+FixShootingOverlay = defclass(FixShootingOverlay, overlay.OverlayWidget)
+FixShootingOverlay.ATTRS{
+    desc='Fixes Shooting your weapon breaking your Throw action.',
+    default_enabled=true,
+    viewscreens={
+        'dungeonmode/Inventory',
+        'dungeonmode/AimProjectile'
+    },
+    frame={w=0, h=0},
+}
+
+function FixShootingOverlay:onInput()
```

Comment:
did you test that this works as expected? I *think* it should, but it definitely needs testing

Reply:
Yep, confirmed